packages/synapse-react-client/src/components/AccessRequirementAclEditor/AccessRequirementAclEditor.tsx

```diff
@@ -166,7 +166,9 @@ export const AccessRequirementAclEditor = React.forwardRef(
           isLoading={isLoadingOriginalAcl}
           isInEditMode={true}
           emptyText={EMPTY_RESOURCE_ACCESS_LIST_TEXT}
-          addResourceAccessItem={addResourceAccessItem}
+          onAddPrincipalToAcl={id =>
+            addResourceAccessItem(id, [ACCESS_TYPE.REVIEW_SUBMISSIONS])
+          }
```

Comment:
change this callback so each implementation of the AclEditor can supply its own default access type (e.g. for ARs, the default resource access set is `['REVIEW_SUBMISSIONS']`, for non-public usergroups on entities it will be `['READ', 'DOWNLOAD']`)